source/agora/test/Base.d

```diff
@@ -939,23 +949,23 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
 
 /*******************************************************************************
 
-    Generate a genesis block given the set of node configurations.
+    Generate a genesis block.
 
-    For each Validator in the node configs, a Freeze TX and an Enrollment
+    For each key pair, a freeze transaction and an Enrollment
     will be created and added to the generated Genesis block.
 
-    The very first transaction is just a payment TX that uses the
+    The very first transaction is just a payment tx that uses the
     keypair in getGenesisKeyPair().
 
     Params:
-        node_configs = the list of node configurations
+        key_pairs = key pairs for signing enrollments with
 
     Returns:
         The genesis block
 
 *******************************************************************************/
 
-private immutable(Block) makeGenesisBlock (in NodeConfig[] node_configs)
+private immutable(Block) makeGenesisBlock (in KeyPair[] key_pairs)
```

Comment:
This change should be moved to the previous commit where the function was introduced.